package.json

```diff
@@ -3,8 +3,7 @@
   "version": "0.0.8",
   "type": "module",
   "scripts": {
-    "test": "node spec/all.spec.js",
-    "dev": "./bin/run.js --port=8000"
```

Comment:
I think the `dev` command may as well stay in

Reply:
For me, the existence of a `dev` command implies that there's auto-reloading of some kind, and I'm not sure why the default port is being overridden as this project only refers to itself running on port `9999` (which is then overridable by whoever runs the project).

I think this is more confusing than it not being there.  There's already `npm start` which is the default for any Node app.